examples/stable_diffusion/openvino/config_unet.json

```diff
@@ -0,0 +1,38 @@
+{
+    "input_model": {
+        "type": "PyTorchModel",
+        "config": {
+            "model_path": "runwayml/stable-diffusion-v1-5",
+            "model_loader": "unet_load",
+            "model_script": "user_script.py"
+        }
+    },
+    "systems": {
+        "local_system": {
+            "type": "LocalSystem",
+            "config": {
+                "accelerators": ["cpu"]
+            }
+        }
+    },
+    "passes": {
+        "conversion": {
+            "type": "OpenVINOConversion",
+            "config": {
+                "user_script": "user_script.py",
+                "example_input_func": "get_unet_example_input",
+                "input": "get_unet_input_shape",
+                "output_model": "unet"
+            }
+        }
+    },
+    "engine": {
+        "search_strategy": false,
+        "evaluate_input_model": false,
+        "host": "local_system",
```

Comment:
nit: both the host and local system are redundant since it's the default.

Reply:
Yeah since this is an example for user, I would like to make it clear.